src/plasoscaffolder/bll/services/file_handler.py

```diff
@@ -12,7 +12,7 @@ def __init__(self):
     """ Constructing the FileHandler"""
 
   @staticmethod
-  def create_file_path(path, name, suffix):
+  def create_file_path(path: os.path, name: str, suffix: str) -> os.path:
     """ Creates the file path out of the directory path, filename and suffix."""
     return os.path.join(path, name + "." + suffix)
```

Comment:
I suggest having as little nested "function" calls as possible, which is easier for troubleshooting
```
return os.path.join(path, name + "." + suffix)
```
this will raise if `name + "." + suffix` fails

so maybe instead:
```
filename = "{0:s}.{1:s}".format(name, suffix)
return os.path.join(path, filename)
```

or:
```
filename = ".".join([name, suffix])
return os.path.join(path, filename)
```